### ISSUE TITLE
Font size of text at the bottom of the homepage needs to be minimum 16 px

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#3991]
This text is currently 14px and needs to be 16px. 

Sinopia is a project of [Linked Data for Production 2 (LD4P2)](https://searchworks.stanford.edu/view/qt361vf0063), generously funded by the Andrew W. Mellon Foundation. All metadata available on Sinopia are published free of restrictions, under the terms of the [Creative Commons CC0 1.0 Universal Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/). Published [Vocabulary](https://sinopia.io/vocabulary).   If you cannot access content or use features on this website due to a disability, please [report your accessibility issue](mailto:library-accessibility-contact@lists.stanford.edu).  Sinopia v3.17.150
